sdk/angelscript/projects/cmake/CMakeLists.txt

```diff
@@ -178,37 +178,38 @@ install(TARGETS ${ANGELSCRIPT_LIBRARY_NAME} EXPORT AngelscriptTargets
 
 install(FILES
 	${CMAKE_CURRENT_SOURCE_DIR}/../../include/angelscript.h
-	DESTINATION include
+	DESTINATION include/${ANGELSCRIPT_LIBRARY_NAME}
 	COMPONENT Devel
 )
 
+
 include(CMakePackageConfigHelpers)
 write_basic_package_version_file(
-	"${CMAKE_CURRENT_BINARY_DIR}/Angelscript/AngelscriptConfigVersion.cmake"
+	"${CMAKE_CURRENT_BINARY_DIR}/Angelscript/${ANGELSCRIPT_LIBRARY_NAME}ConfigVersion.cmake"
 	VERSION ${PROJECT_VERSION}
 	COMPATIBILITY AnyNewerVersion
 )
 
 export(EXPORT AngelscriptTargets
-	FILE "${CMAKE_CURRENT_BINARY_DIR}/Angelscript/AngelscriptTargets.cmake"
-	NAMESPACE Angelscript::
+	FILE "${CMAKE_CURRENT_BINARY_DIR}/Angelscript/${ANGELSCRIPT_LIBRARY_NAME}Targets.cmake"
+	NAMESPACE AngelScript::
 )
-configure_file(cmake/AngelscriptConfig.cmake
-	"${CMAKE_CURRENT_BINARY_DIR}/Angelscript/AngelscriptConfig.cmake"
-	COPYONLY
+
+set(TARGETS_EXPORT_NAME "${ANGELSCRIPT_LIBRARY_NAME}Targets")
+configure_package_config_file(
+    cmake/Config.cmake.in
+    "${CMAKE_CURRENT_BINARY_DIR}/Angelscript/${ANGELSCRIPT_LIBRARY_NAME}Config.cmake"
+    INSTALL_DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/Angelscript/${ANGELSCRIPT_LIBRARY_NAME}Config.cmake"
 )
 
 set(ConfigPackageLocation lib/cmake/Angelscript)
 install(EXPORT AngelscriptTargets
-	FILE AngelscriptTargets.cmake
-	NAMESPACE Angelscript::
```

Comment:
I don't think that changing that is a good idea, it's a public CMake API, similar to public C++ API. If you want to change it, please discuss it with upstream developers.

Reply:
Changing what? Hunter refers to angelscript targets by "AngelScript::" so that's wrong in the first place if we are to keep the namespace intact. Again I have no idea what am I uspposed to "bend" - Hunters approach or original CMakeFiles because they just don't work together "as is"